pkg/combustion/templates/15-rke2-installer.sh.tpl

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+set -euo pipefail
+
+mount /var
+mkdir -p /var/lib/rancher/rke2/agent/images/
+cp {{ .ImagesPath }}/* /var/lib/rancher/rke2/agent/images/
+umount /var
+
+{{- if .ConfigFile }}
+mkdir -p /etc/rancher/rke2/
+cp {{ .ConfigFile }} /etc/rancher/rke2/config.yaml
+{{- end }}
+
+{{- if .NodeType }}
+export INSTALL_RKE2_TYPE={{ .NodeType }}
+{{- end }}
+
+export INSTALL_RKE2_TAR_PREFIX=/opt/rke2
+export INSTALL_RKE2_ARTIFACT_PATH={{ .InstallPath }}
+
+./rke2_installer.sh
+
+echo "export KUBECONFIG=/etc/rancher/rke2/rke2.yaml" >> ~/.bashrc
+echo "export PATH=${PATH}:/var/lib/rancher/rke2/bin" >> ~/.bashrc
```

Comment:
What's the purpose for adding this to `PATH`? Is it just to get access to `kubectl` when you log into the box?

If this isn't needed for RKE2 to actually run in the first place, I think we should at least consider removing this. Generally speaking, production machines are locked down to all hell, so having this in `PATH` may be frowned upon by the more anal-retentive security admins.

Adding @rdoxenham for both his input based on his Linux experience and ultimately his recommendation as PM.

Reply:
I indeed use it for the `kubectl` usages and I guess we can live without it. Makes the development stage just slightly easier though. I'm fine with dropping it.

Reply:
I would suggest removing it from the code but adding the steps in a "setups for development" best practices doc

Reply:
Agreed, we should drop this from EIB. Interestingly, the k3s installer does this automagically, but rke2 doesn't. Either way, this should be a post-deployment action as/when/if required, not something we do by default.

Reply:
I've asked about it before and was told K3s does this by default because it's more likely used in local dev environment.